core/src/main/resources/commonsdivision/article/index.scala.html

```diff
@@ -1,5 +1,7 @@
 @import com.gu.contentatom.renderer.utils.GuardianDateFormatter
 @import com.gu.contentatom.thrift.atom.commonsdivision.Mp
+@import scala.collection.immutable.Map
+@import scala.collection.immutable.Map
```

Comment:
These look the same to me. Am I missing something 🤔

Reply:
woops